service/src/main/java/dk/dbc/search/solrdocstore/BibliographicToBibliographicEntity.java

```diff
@@ -0,0 +1,14 @@
+package dk.dbc.search.solrdocstore;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "BibliographicToBibliographic")
+public class BibliographicToBibliographicEntity {
+    @Id
+    public String oldRecordId;
+
+    public String newRecordId;
+}
```

Comment:
Jg bryder mig ikke specient om old/new recordid, det giver opfattelsen af dette er en del af det drivende for at få ændret en recordid ikke at den allerede er ændret.
Jeg ville forterække superceded/decommissionedRecordId og actual/real/currentRecordId